yapapi/payload/vm.py

```diff
@@ -80,6 +90,7 @@ async def repo(
     min_mem_gib: float = 0.5,
     min_storage_gib: float = 2.0,
     min_cpu_threads: int = 1,
+    capabilities: Optional[List[VmCaps]] = None,
```

Comment:
Do I correctly understand that if we have more than one element on the list than **all** capabilities should be offered?
And if yes, why not just and empty list here? IOW, if empty list is the same as `None`, I'd rather have only one option.

Reply:
(Not an empty list, but `field(default_factory=list)` i think)

Reply:
I think `field(default_factory=list)` cannot be a _value_ of a field...

Reply:
OK, sorry, I thought this is a dataclass